helm/cluster-cloud-director/templates/_helpers.tpl

```diff
@@ -159,7 +159,7 @@ template: {{ .currentClass.template }}
 sizingPolicy: {{ .currentClass.sizingPolicy }}
 placementPolicy: {{ .currentClass.placementPolicy }}
 storageProfile: {{ .currentClass.storageProfile }}
-diskSize: {{ .currentClass.diskSize }}
+diskSize: {{ mul .currentClass.diskSizeGB 1024 1024 1024}}
```

Comment:
We want the value to be interpreted as metric [gigabyte](https://en.wikipedia.org/wiki/Gigabyte) (1000^3), not as IEC gibibyte (1024^3).

```suggestion
diskSize: {{ mul .currentClass.diskSizeGB 1000 1000 1000}}
```

Reply:
If I use 1000 it creates a disk of 27.94 GB instead of 30 GB.